templates/job_upstart.erb

```diff
@@ -1,4 +1,4 @@
 # FLAG: MANAGED BY PUPPET
 start on <%= @reporting_name %>
 task
-exec setuidgid '<%= @user %>' '/nail/etc/upstart_crons/<%= @title %>'
```

Comment:
Is this right? Does it not actually go into the job_cron template so that the wrapper is invoked by cron, and not by upstart?

Or is the purpose of having it in the upstart job so that it gets "unstale" on boot or upon exec?